babashka.el

```diff
@@ -37,8 +37,34 @@
 ;;
 ;;; Code:
 (require 'parseedn)
+(require 'seq)
 (require 'project)
 
+(defgroup babashka nil
+  "Babashka Tasks Interface"
+  :group 'external)
+
+(defcustom babashka-async-shell-command #'async-shell-command
```

Comment:
in theory, we can use this to run tasks outside of Emacs as well, like terminal emulators, tmux etc